src/abstractdataframe/io.jl

```diff
@@ -101,7 +109,8 @@ function writetable(filename::String,
                    df,
                    header = header,
                    separator = separator,
-                   quotemark = quotemark)
+                   quotemark = quotemark,
+				   nastring = nastring)
     end
```

Comment:
Looks like tabs scattered throughout the code? Matching the 4-space indent convention everywhere should clear up any alignment issues.